### PR TITLE
Store registry data on Persistent Volume

### PR DESCRIPTION
**Pull request by novakg**
Storing docker registry's data on PV as discussed in https://github.com/fabric8io/fabric8/issues/6688#issuecomment-270776017

### CODE REVIEW

File: fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fabric8-docker-registry-storage
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```
Comment (rawlingsj) [inline fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml]: I wonder if we should increase this to say 5Gi?

**Comment by rawlingsj** [comment]
This is great - thanks @novakg!  Just the one comment about the storage request size to address and we can merge this.

File: fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml (hunk 1 above)
Comment (novakg) [inline fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml]: Well, personally I would use 100Gi for all PVs in Fabric8, but that depends on the needs of each user. Might be configurable through gofabric8's parameters, but that's a feature for the future I think :) I change it to 5Gi for now.

Comment (rawlingsj) [inline fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml]: Agreed - raised https://github.com/fabric8io/fabric8/issues/6694 to track the configureable / editable storage requests.

**Pull request merged**